jujugui/static/gui/src/app/views/utils.js

```diff
@@ -1330,7 +1330,8 @@ YUI.add('juju-view-utils', function(Y) {
         navbar = Y.one('.header-banner'),
         viewport = Y.one('#viewport'),
         result = {height: minheight || 0, width: minwidth || 0};
-    if (containerHeight && navbar && viewport) {
+    if (containerHeight && navbar && viewport &&
+      viewport.getComputedStyle('width') !== 'auto') {
```

Comment:
So it was getting an error because it sets the height and width based on the viewport. But when an element is set to `display: none` it sets the width to auto. This was resulting in a NaN further down in this statement when the width value is computed.